CGPs/0023.md

```diff
@@ -10,43 +10,42 @@
 
 This governance proposal enables oracle addresses to make them able to report the value of CELO/EUR for the cEUR contract, deployed on [CR3](https://github.com/celo-org/celo-proposals/blob/master/CGPs/0022.md). It also proposed to send 100 CELO to each of the oracles to cover gas fees for the first couple of years and freezing StableTokenEUR (cEUR) and ExchangeEUR.
```

Comment:
I think `It also proposed` -> `It also proposes`

Reply:
Not super passionate but it may be good to rationalize the 100 CELO amount, eg how much we've seen existing oracles use and why 100 makes sense